t/mojo/request_cgi.t

```diff
@@ -291,6 +291,38 @@ subtest 'Parse Apache 2.2.14 CGI environment variables and body (root)' => sub {
   is $req->url->to_abs->to_string, 'http://127.0.0.1:13028/upload/', 'right absolute URL';
 };
 
+subtest 'SERVER_PORT (spec) superior to HTTP_HOST (not-spec)' => sub {
```

Comment:
With what kind of configuration would Apache/2.2.14 have different values in `SERVER_PORT` and `HTTP_HOST`?

Reply:
SetEnv SERVER_PORT 8080
SetEnv HTTP_HOST "http://google.com:80"

But I'm not using Apache, so not sure. Using a different webserver.